public/scripts/create.js

```diff
@@ -17,9 +17,11 @@ function changePopupDisplay(popup) {
 }
 
 function getInput() {
-  const input = document.getElementById('width-height');
+  let Data = [];
```

Comment:
it should be data, without capital letter

Reply:
Thanks, I`ll change it in the next pull request